src/uu/fmt/src/fmt.rs

```diff
@@ -155,7 +169,10 @@ fn parse_arguments(args: impl uucore::Args) -> UResult<(Vec<String>, FmtOptions)
             }
         };
         if !matches.get_flag(OPT_WIDTH) {
-            fmt_opts.width = cmp::max(fmt_opts.goal * 100 / 94, fmt_opts.goal + 3);
+            fmt_opts.width = cmp::max(
```

Comment:
This doesn't really have anything to do with your change, but are we sure this is right?

I also think the way this is implemented is a bit strange, maybe it would be better in a match expression:

```rust
let width_opt = matches.get_one::<String>(OPT_WIDTH);
let goal_opt = matches.get_one::<String>(OPT_GOAL);
match (width_opt, goal_opt) {
    (Some(width), Some(goal)) => {...},
    ...
}
```

Maybe we can make a good-first-issue for someone to clean this up a bit.